<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMenuAndButtonsWidget</class>
 <widget class="QWidget" name="DialogMenuAndButtonsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>76</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="mMenuAndButtonsLayout">
     <item row="1" column="0">
      <spacer name="mButtonsSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="mOkay">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>This allows the user to apply changes and close the dialog.</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="mReset">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>This allows the user to reset the geometric transformation of the ghost scenario/selection of objects.</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="mHelp">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>This displays the brief help text for the dialog.</string>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="mApply">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>This allows the user to apply the geometric transformation to the real scenario/selection of objects.</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="6">
      <widget class="UtQtComboBox" name="mMenu">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>This allows the user to switch modes in the dialog.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="mCancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtComboBox</class>
   <extends>QComboBox</extends>
   <header>UtQtComboBox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
